EssentialFeed/EssentialFeediOS/Image Comments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController: NSObject, UITableViewDataSource {
+	let model: ImageCommentViewModel
```

Comment:
This instance property should not be accessible, thus it should be marked as `private`:
```suggestion
	private let model: ImageCommentViewModel
```